torch/ao/quantization/fake_quantize.py

```diff
@@ -126,8 +127,12 @@ def __init__(self, observer=MovingAverageMinMaxObserver, quant_min=0, quant_max=
         self.activation_post_process = observer(**observer_kwargs)
         assert torch.iinfo(self.activation_post_process.dtype).min <= quant_min, 'quant_min out of bound'
         assert quant_max <= torch.iinfo(self.activation_post_process.dtype).max, 'quant_max out of bound'
+        if(self.activation_post_process.qscheme == torch.per_channel_affine_float_qparams):
```

Comment:
would it work to get the dtype from `activation_post_process.zero_point.dtype`?

Reply:
If `activation_post_process` is an Observer, it will not have `zero_point` as a member, since this is calculated in `_calculate_qparams`
see: https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/observer.py#L283-#L325.

To clean this up, I added a a helper `_is_float_qparams`, similar to the other `qscheme` helper functions.
see: https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/fake_quantize.py#L20